lib/two_factor_authentication/models/two_factor_authenticatable.rb

```diff
@@ -46,6 +67,18 @@ def need_two_factor_authentication?(request)
           true
         end
 
+        def otp_code
+          direct_otp
```

Comment:
I left otp_code here only for backward compat.   If you are ok with breaking that I will just drop this method completely.